lang/smlnj/Portfile

```diff
@@ -64,8 +64,7 @@ extract.mkdir       yes
 extract.only        config.tgz
 
 ### patch ###
-patchfiles          patch-config__install.sh-correct-compiler.diff \
-                    patch-_arch.diff
```

Comment:
For anyone wondering, removing this patchfile is correct because the change was included upstream in v110.99.5:

https://github.com/smlnj/legacy/commit/24b3bc2886848d27afb9fcd145f2803925069162